### DISCUSSION TITLE
Offline Cache

### DISCUSSION THREAD

**Discussion by RagingFlames** [Docile-Alligator/Infinity-For-Reddit#881] [Ideas]
I've moved to infinity from Slide because it is slowly breaking down and not being maintained. One of the features of slide that I really like and used often was it's offline cache option. Every morning at like, 7 AM, I configured slide to cache a bunch of posts and their comments from a tonne of subreddits I selected. That way when I go somewhere with no internet, like a plane, I still have entertainment. It would be nice to see this feature in Infinity too.

**Comment by jakenvac** [comment]
I would love to see this. prefetch even while scrolling would be nice, too!
